pkgs/stdenv/generic/setup.sh

```diff
@@ -1351,9 +1351,9 @@ checkPhase() {
     if [[ -z "${checkTarget:-}" ]]; then
         #TODO(@oxij): should flagsArray influence make -n?
         if make -n ${makefile:+-f $makefile} check >/dev/null 2>&1; then
-            checkTarget=check
+            checkTarget="check"
```

Comment:
This hunk (and the next) logically belong to the other commit (about the quoting).

Reply:
Addressed.